actions/src/main/res/values/strings.xml

```diff
@@ -58,4 +58,7 @@
     <string name="action_upload_queue_subtitle">Uploading in queue. Please track the status in the mobile app.</string>
     <string name="action_upload_queue_ok_button">Ok</string>
 
+    <string name="share_files_permissions_rationale">@string/share_files_failure_permissions</string>
+    <string name="share_files_failure_permissions">To share files, turn on Read storage permissions.</string>
```

Comment:
Correct the grammar. Read should be read.

Reply:
Done